### DISCUSSION TITLE
Upon Copy, optionally include the File Name

### DISCUSSION THREAD

**Discussion by MikeSeim** [mvdkwast/obsidian-copy-as-html#34] [Ideas]
I think it'd be really nice to have the option to include the title of the file, the file name, included at the first line of the copied content as Heading-1-styling. If included, perhaps include two line breaks to make it format nicely.

I imagine some folks include the title of their note on the first line in the note whereas others might use the filename as the title.  For those in the later camp, like me, that'd be really nice.

I found this plugin today for my recipe vault. When I want to print the recipe, I find Obsidian's export to PDF feature putzy. This plugin actually makes it easier; I think.  I just "ALT+P" and paste in a blank work document. Super nice! I can even quickly delete aspects of my recipe I don't want printed, notes for example. But I'm missing the recipe title which is actually the file name.

**Comment by mvdkwast** [comment]
Hello @MikeSeim , thanks for your idea and interesting use-case for this plugin !

I will create an enhancement request for this.

**Comment by mvdkwast** [comment]
Released in v0.6.0

**Reply by MikeSeim** [reply]
Outstanding! Thank you very much.
